packages/card/src/Card.story.tsx

```diff
@@ -5,7 +5,7 @@ import { Story } from '@storybook/react';
 import { BoxProps } from '@leafygreen-ui/box';
 import { storybookArgTypes } from '@leafygreen-ui/lib';
 
-import { Card as LGCard, CardProps } from './Card';
+import { Card as LGCard, CardProps } from './Card/Card';
```

Comment:
Looks like the subdir is missing an index

Reply:
good catch